docs/src/main/sphinx/release/release-397.md

```diff
@@ -0,0 +1,46 @@
+# Release 397 (21 Sep 2022)
+
+## General
+
+* Fix incorrect parsing of invalid values in cast from `varchar` to `timestamp`. ({issue}`14164`)
+* Fix potential incorrect results for queries where all data scanned from tables
```

Comment:
not sure what "potential" incorrect results is supposed to indicate ...

also dont know if we need to explain leaf stage and downstream more or somehow make this more consumable.

Reply:
We don't need to provide those details. If someone wants to find out more, they can look at the PR. However, the RN entry should explain the conditions under which the correctness issue might happen (type of queries, etc)

Reply:
My understanding was that this was the condition in which it happened. Let me try again with info from the PR, may need another suggestion/edit.